src/config/ConfigManager.cpp

```diff
@@ -633,6 +633,26 @@ std::string CConfigManager::getMainConfigPath() {
     return getConfigDir() + "/hypr/" + (ISDEBUG ? "hyprlandd.conf" : "hyprland.conf");
 }
 
+const std::string CConfigManager::getConfigString() {
+    std::string configString;
+    std::string line;
+
+    for (auto path : this->configPaths) {
+        std::ifstream configFile(path.c_str());
+        configString += ("\n\nConfig File: " + path + ": ");
+        if (!configFile.is_open()) {
+            Debug::log(LOG, "Config file not readable/found!");
+            configString += "Read Failed\n";
+            continue;
+        }
+        configString += "Read Succeeded\n";
+        while (std::getline(configFile, line)) {
```

Comment:
brudda
```cpp
std::string content(std::istreambuf_iterator<char>(ifs), std::istreambuf_iterator<char>());
```

Reply:
Lol I was looking for this in the standard library. ty for pointing this out.